data/tutorials/lg_08_error_handling.md

```diff
@@ -660,17 +663,17 @@ val Option.to_result : none:'e -> 'a option -> ('a, 'e) result
 
 - `File.read_opt` is supposed to open a file, read its contents and return it as
 a string wrapped in an option, if anything goes wrong `None` is returned.
-- `Yaml.of_string` parses a string an turns into an ad-hoc OCaml type
-- `Yaml.find` recursively searches a key in a Yaml tree, if found, it returns
+- `Yaml.of_string` parses a string and turns it into an ad hoc OCaml type
+- `Yaml.find` recursively searches for a key in a Yaml tree. If found, it returns
   the corresponding data, wrapped in an option
 - `Option.to_result` performs conversion of an **`option`** into a **`result`**.
 - Finally, `let*` stands for `Result.bind`.
 
-Since functions from the `Yaml` module both returns **`result`** data, it is easier
-to write a pipe which processes that type all along. That's why
+Since both functions from the `Yaml` module return **`result`** data, it is
```

Comment:
```suggestion
Since both functions from the `Yaml` module return **`result`** data, it is
```
Unless the actual command is capitalized like `Yaml`, I suggest keeping it all lower case `yaml` (or exactly how it appears in the command). If it's not a command, then it needs to be all capitalized like YAML. Basically match the formatting they use here: https://yaml.org/

Reply:
Good point. Actually, it's above that we need to write YAML. Here, it's a module name, it has to stay like that.

Reply:
Okay, cool. I figured it was a different format since it was a module, but should it then be `yaml` or `Yaml`?

Reply:
That's super tricky. 
- Technology Name: YAML
- OCaml module name : Yaml
- Opam package name: yaml